storage/localstore/mode_put.go

```diff
@@ -221,6 +221,13 @@ func (db *DB) putUpload(batch *leveldb.Batch, binIDs map[uint8]uint64, item shed
 	if exists {
 		return true, nil
 	}
+	anonymous := false
+	if db.tags != nil {
+		tag, err := db.tags.Get(item.Tag)
+		if err == nil {
```

Comment:
Not so correct to ignore error value. Anonymous tag should be set only if tag is not found, and error returned if it is not nil, even currently there is only tag not found error that can be returned here, but that is an implementation detail of tags that can change and allow incorrect logic here.